### PR TITLE
Add shields to the README

### PR DESCRIPTION
**Pull request by LeonMatthesKDAB**
Should make our repository a bit more colorful and engaging.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,16 @@ SPDX-License-Identifier: MIT OR Apache-2.0
...
 
+[![Github](https://img.shields.io/badge/github-kdab%2Fcxx--qt-informational?logo=github)](https://github.com/kdab/cxx-qt)
+[![Book status](https://img.shields.io/github/actions/workflow/status/kdab/cxx-qt/book.yml?label=book&logo=mdbook)](https://kdab.github.io/cxx-qt/book)
+![GitHub Workflow Status](https://img.shields.io/github/actions/workflow/status/kdab/cxx-qt/github-cxx-qt-tests.yml)
+![License (MIT/Apache2.0)](https://img.shields.io/crates/l/cxx-qt)
+| Crate | Status |
+| --- | --- |
+| cxx-qt | [![docs.rs](https://img.shields.io/docsrs/cxx-qt?logo=docsdotrs)](https://docs.rs/cxx-qt) [![Crates.io](https://img.shields.io/crates/v/cxx-qt)](https://crates.io/crates/cxx-qt) |
+| cxx-qt-build | [![docs.rs](https://img.shields.io/docsrs/cxx-qt-build?logo=docsdotrs)](https://docs.rs/cxx-qt-build) [![Crates.io](https://img.shields.io/crates/v/cxx-qt-build)](https://crates.io/crates/cxx-qt-build) |
+| cxx-qt-lib | [![docs.rs](https://img.shields.io/docsrs/cxx-qt-lib?logo=docsdotrs)](https://docs.rs/cxx-qt-lib) [![Crates.io](https://img.shields.io/crates/v/cxx-qt-lib)](https://crates.io/crates/cxx-qt-lib) |
```
[the remark this answers is not in the document]
Comment (LeonMatthesKDAB) [reply README.md:20]: Can do, wasn't sure whether we want to publicly promote it.
That's why I left out cxx-qt-gen as well.

But qt-build-utils is more user-facing I guess :thinking:

[the remark this answers is not in the document]
Comment (LeonMatthesKDAB) [reply README.md:20]: Added :blush:

File: README.md [hunk 2]
```diff
@@ -9,6 +9,17 @@ SPDX-License-Identifier: MIT OR Apache-2.0
 
 # CXX-Qt
```
Comment (ahayzen-kdab) [inline README.md:11]: could we add `[REUSE status](https://api.reuse.software/info/github.com/KDAB/cxx-qt)` as well like Slint has? :-)

Comment (LeonMatthesKDAB) [reply README.md:11]: Great suggestion. Done

**Pull request merged**